example.js

```diff
@@ -89,11 +94,11 @@ define([
 				contentVideoEl : $("#contentVideo")[0],
 				bandWidth : $("#videoBw").prop('value'),
 				devices   : BJN.localDevices,
-				evtVideoUnmute : unmuteVideo,
```

Comment:
We're not using this callback in Virtual Rooms and supporting it will take a bit of work.

`evtVideoUnmute` is called immediately after a connection has been established with the webcam (more info [here](https://github.com/bluejeans/sdk-webrtc-meetings#-webrtc-client-sdk-event-callbacks)).

When the BJN client is initialized with video premuted, this function is still called, and incorrectly updates the UI to indicate that the video has been unmuted.